lib/features/children/generosity_challenge_chat/chat_scripts/models/enums/chat_script_function.dart

```diff
@@ -38,4 +41,12 @@ enum ChatScriptFunction {
     }
     return true;
   }
+
+  static Future<bool> _registerUser(
+    BuildContext context,
+  ) async {
+    // KIDS-941: Implement registration
+    throw UnimplementedError();
```

Comment:
Are you sure you want this PR merged into develop when it throws this error here?

Reply:
Yeah, people will not get to that day soon :) so it will not get trigger